src/components/navbar.vue

```diff
@@ -1,19 +1,36 @@
 <template>
   <nav>
     <h1 class="title">Polling System</h1>
-    <div class="home">
-      <router-link to="/">Home</router-link>
+    <div class="home" v-if="user">
+      <router-link to="/home">Logged in as {{ user.username }}</router-link>
+      <button @click.prevent="logout">logout</button>
     </div>
-    <div class="logins">
-      <router-link to="/login">Login</router-link>
+    <div class="logins" v-if="!user">
+      <router-link to="/">Login</router-link>
       <router-link to="/signin">Signin</router-link>
     </div>
   </nav>
 </template>
 
 <script>
+import { loginApi } from "../composables/loginApi.js";
 export default {
   name: "navBar",
-  setup() {},
+  setup() {
+    const { user, logout } = loginApi();
+
+    return { user, logout };
```

Comment:
if this is the case 
const { user, logout } = loginApi();
    return { user, logout };
    and you are not using any function from composable api, you can directly return it like
    
    return {
    ...loginApi()
    }